lib/flutter_local_notifications.dart

```diff
@@ -100,6 +114,23 @@ class FlutterLocalNotificationsPlugin {
     });
   }
 
+  Future showDailyAtTime(int id, String title, String body,
+    Time notificationTime, NotificationDetails notificationDetails,
+    {String payload}) async {
+    Map<String, dynamic> serializedPlatformSpecifics =
+        _retrievePlatformSpecificNotificationDetails(notificationDetails);
+    await _channel.invokeMethod('periodicallyShow', <String, dynamic>{
```

Comment:
invokeMethod should pass in 'showDailyAtTime' for better separation. This means that the onMethodCall override on the native side will need a different case to handle what needs to be done. If you could also split up repeatNotification to two separate methods (one for interval-based, the other for time-based) then that would be good as well

Reply:
I have chosen this approach because I considered this feature as a Daily `periodicallyShow` use case, that way changes in both Java and Objective-C remain minimal. I can add that change though.